blog/2024-03-20-state_reported_timestamp.md

```diff
@@ -0,0 +1,29 @@
+---
+author: Erik Montnemery
+authorURL: https://github.com/emontnemery
+title: "New state timestamp State.last_reported"
+---
+
+The `State` object is now always updated and an event is always fired when an integration sets the state of an entity, regardless of any change to the state or a state attribute. This is implemented by adding a new timestamp, `State.last_reported` and a new event `state_reported`.
```

Comment:
We should also document the state object attributes somewhere in the dev docs.

Reply:
The developer documentation has a link to the user documentation <https://www.home-assistant.io/docs/configuration/state_object/>, which should indeed also be updated 👍